root/static/styles/extra/banner.less

```diff
@@ -1,9 +1,10 @@
 .banner {
     margin: @form-margin 0;
+    margin-left: 12px;
+    margin-right: 12px;
     text-align: center;
-    background: #FFFFFF;
+    border: #CCC 1px solid;
```

Comment:
`@medium-border`

I think the banners look a lot better with `background: @effectively-white-bg` too, instead of just a border